test/mount_utils_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright Modal Labs 2024
+import pytest
+from collections.abc import Mapping
+
+from modal._utils.mount_utils import validate_mount_points, validate_network_file_systems, validate_volumes
+from modal.exception import InvalidError
+from modal.network_file_system import _NetworkFileSystem
+from modal.volume import _Volume
+
+
+def test_validate_mount_points():
+    # valid mount points
+    dict_input: Mapping = {"/foo/bar": _NetworkFileSystem.from_name("NetworkFileSystem", create_if_missing=False)}
```

Comment:
Odd that we need the explicit type annotation here? A dict should be accepted by an interface typed with `Mapping`.

Reply:
I agree but without it I'm getting

```
error: Argument 2 to "validate_mount_points" has incompatible type "dict[str, NetworkFileSystem]"; expected "Mapping[Union[str, PurePosixPath], NetworkFileSystem]"
```

which doesn't make sense to me